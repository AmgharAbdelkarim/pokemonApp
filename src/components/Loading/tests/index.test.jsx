import { cleanup, render } from "@testing-library/react";

import LoadingState from "components/Loading";

afterEach(cleanup);

describe("LoadingState", () => {
  it("should LoadingState render correctly", () => {
    const { asFragment, getByText } = render(LoadingState());
    expect(getByText("Loading...")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
