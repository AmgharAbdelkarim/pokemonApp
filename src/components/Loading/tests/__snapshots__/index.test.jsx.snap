// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingState should LoadingState render correctly 1`] = `
<DocumentFragment>
  <h3>
    Loading...
  </h3>
</DocumentFragment>
`;
